<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="plot_1" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>5</horstretch>
      <verstretch>5</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plot_2" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>5</horstretch>
      <verstretch>5</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="button_view_1">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>20</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>View 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_bar_2">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>140</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Bar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_line_2">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>110</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_line_1">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>50</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_bar_1">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>80</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Bar</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plot_3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>325</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>5</horstretch>
      <verstretch>5</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plot_4" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>325</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>5</horstretch>
      <verstretch>5</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="button_bar_3">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>200</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Bar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_line_3">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>170</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_bar_4">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>260</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Bar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_line_4">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>230</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_view_2">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>50</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>View 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_view_3">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>80</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>View 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_view_4">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>110</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>View 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save_2">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>320</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save_1">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>290</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save_4">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>380</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save_3">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>350</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Filename">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>210</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_search">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>240</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchField">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>170</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>150</y>
      <width>100</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Word</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>190</y>
      <width>100</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Filename</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>400</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_MPA">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>460</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add MPA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_MPA">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>460</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="toggle_legend">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>300</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Hide Legend</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>600</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNope">
    <property name="title">
     <string>Nope</string>
    </property>
   </widget>
   <addaction name="menuNope"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
